src/video/gl/gl_texture.cpp

```diff
@@ -125,7 +125,8 @@ GLTexture::GLTexture(const SDL_Surface& image, const Sampler& sampler) :
       const int bpp = convert->format->BytesPerPixel;
       const int x = m_image_width - 1;
       const int y = m_image_height - 1;
-      Uint32 color = *reinterpret_cast<Uint32*>(static_cast<uint8_t*>(convert->pixels) + y * convert->pitch + x * bpp);
+      Uint32 color = 0;
+      memcpy(&color, static_cast<uint8_t*>(convert->pixels) + y * convert->pitch + x * bpp, bpp);
```

Comment:
Is this related? If so, approved.

Reply:
This is to fix a warning with the OpenGL renderer on Ubuntu Touch (and possibly other older OSes as well).